### ISSUE TITLE
How do it make it stop from installing automatically in Atom?

### ISSUE THREAD

**Issue by bernyosuna** [atom-community/atom-ide-code-format#15]
No matter what, every time I save my code the way I have it and I want it, this package modifies it as whatever he thinks it should be and I just hate it. Even if I delete this package it automatically installs on start.

**Comment by UziTech** [comment]
You should be able to uninstall it from the settings.

**Comment by bernyosuna** [comment]
Yes, I know I can uninstall them, but every time I start Atom it just re-installs without even asking

**Comment by UziTech** [comment]
You probably have another package that is installing it automatically. Can you give a list of installed packages?

**Comment by aminya** [comment]
Disable automatic formatting from the settings
